<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="claudiupc" ignored="1" name="Helioscript-SmokeTests(FIREFOX)" tests="12" failures="3" timestamp="11 Feb 2020 12:33:38 GMT" time="324.427" errors="0">
  <testcase name="add_new_coupon" time="5.229" classname="tests.SmokeTests"/>
  <testcase name="adjustAdd_inventory" time="8.251" classname="tests.SmokeTests"/>
  <testcase name="adjustRemove_inventory" time="8.644" classname="tests.SmokeTests"/>
  <testcase name="invalidCredentials" time="1.224" classname="tests.SmokeTests"/>
  <testcase name="receive_inventory" time="8.231" classname="tests.SmokeTests"/>
  <testcase name="shipping_method" time="11.639" classname="tests.SmokeTests"/>
  <testcase name="setCommission" time="7.468" classname="tests.SmokeTests"/>
  <testcase name="medium_Management" time="7.459" classname="tests.SmokeTests"/>
  <testcase name="addNewPatient" time="16.345" classname="tests.SmokeTests"/>
  <testcase name="send_AdminInvitation" time="8.266" classname="tests.SmokeTests">
    <failure type="java.lang.AssertionError" message="expected [Invite has been sent.] but found [Employee RORY ISLA already invited.]">
      <![CDATA[java.lang.AssertionError: expected [Invite has been sent.] but found [Employee RORY ISLA already invited.]
at pages.AdminSettings$UserList.assert_sentInvitation(AdminSettings.java:201)
at tests.SmokeTests.send_AdminInvitation(SmokeTests.java:150)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- send_AdminInvitation -->
  <testcase name="send_CompounderInvitation" time="8.419" classname="tests.SmokeTests">
    <failure type="java.lang.AssertionError" message="expected [Invite has been sent.] but found [Employee HANNAH MUHAMMAD already invited.]">
      <![CDATA[java.lang.AssertionError: expected [Invite has been sent.] but found [Employee HANNAH MUHAMMAD already invited.]
at pages.AdminSettings$UserList.assert_sentInvitation(AdminSettings.java:201)
at tests.SmokeTests.send_CompounderInvitation(SmokeTests.java:159)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- send_CompounderInvitation -->
  <testcase name="send_CSRInvitation" time="8.302" classname="tests.SmokeTests">
    <failure type="java.lang.AssertionError" message="expected [Invite has been sent.] but found [Employee EMMA LOUIS already invited.]">
      <![CDATA[java.lang.AssertionError: expected [Invite has been sent.] but found [Employee EMMA LOUIS already invited.]
at pages.AdminSettings$UserList.assert_sentInvitation(AdminSettings.java:201)
at tests.SmokeTests.send_CSRInvitation(SmokeTests.java:168)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- send_CSRInvitation -->
  <testcase name="orderHour" time="0.0" classname="tests.SmokeTests">
    <ignored/>
  </testcase> <!-- orderHour -->
</testsuite> <!-- Helioscript-SmokeTests(FIREFOX) -->
